apps/edgeclk/app.js

```diff
@@ -279,6 +280,10 @@
     drawLower();
   };
 
+  const onStep = function () {
+    if (settings.redrawOnStep) drawSteps();
```

Comment:
Maybe I should do this check before registering the onStep listener instead? Better for battery, but don't know if significant. 

I did this style since I thought it matched the coding style of the app better.

Reply:
Yeah I think we break tradition and do the check before adding/removing the listener, but happy to merge as-is if you prefer

Reply:
I refactored it 👍

Reply:
Nice!